oak_containers_launcher/src/qemu.rs

```diff
@@ -50,6 +50,10 @@ pub struct Params {
     #[arg(long)]
     pub memory_size: Option<String>,
 
+    /// How many CPUs to give to the VM.
```

Comment:
Haha, what happens when `num_cpus` is None? No CPUs for the VM? Maybe 1 as default value? And maybe add in a comment what the largest number is you'd expect work/tested with?